test/src/in_language_tests/test_programs/revert_inline_tests/src/main.sw

```diff
@@ -14,3 +14,8 @@ fn revert_revert_require() {
 fn pass_revert_require() {
     require(true, "error");
 }
+
+#[test(should_revert)]
```

Comment:
Can we assert on the panic message?
```suggestion
#[test(should_revert = "error")]
```

Reply:
tests fail with this, i think this might be checking the revert signal and not the logs